swarm/pss/api.go

```diff
@@ -137,3 +137,13 @@ func (pssapi *API) SendAsym(pubkeyhex string, topic Topic, msg hexutil.Bytes) er
 func (pssapi *API) SendSym(symkeyhex string, topic Topic, msg hexutil.Bytes) error {
 	return pssapi.Pss.SendSym(symkeyhex, topic, msg[:])
 }
+
+func (pssapi *API) GetPeer(pubkeyhex string) ([]Topic, error) {
```

Comment:
if you are returning topics, why is the method called `GetPeer`?
rename to `GetPeerTopics` or `GetTopicsForPeer`

Reply:
also, why not change the signature so you could just return the `pssapi.Pss.GetPeer(pubkeyhex)` directly? (as in `GetPeerAddress`)

Reply:
I think you can only return one param with RPC